traffic_ops/traffic_ops_golang/steeringtargets/steeringtargets.go

```diff
@@ -160,6 +160,10 @@ func read(tx *sqlx.Tx, parameters map[string]string, user *auth.CurrentUser) ([]
 }
 
 func (st *TOSteeringTargetV11) Create() (error, error, int) {
+	valid, err := tc.ValidateTypeID(st.ReqInfo.Tx.Tx, st.TypeID, "steering_target")
```

Comment:
This validation should live in `lib/go-tc/steeringtarget.go:Validate()` -- same w/ the `Update` validation below. Putting it there should cover both cases.